### PR TITLE
update products dan login

### PR DESCRIPTION
**Pull request by RyeConstantine**
update products dan login

### CODE REVIEW

File: pages/services/productServices.js
Comment (field006) [inline pages/services/productServices.js:1]: is this file usedd ?

File: layouts/default1.vue
Comment (field006) [inline layouts/default1.vue:1]: in what pages you use this layout ?

File: layouts/noheader.vue [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<template>
+  <v-app dark>
+    <!--<v-navigation-drawer
```
Comment (field006) [inline layouts/noheader.vue]: please remove unused code

File: nuxt.config.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 
 export default {
   // Disable server-side rendering: https://go.nuxtjs.dev/ssr-mode
   ssr: false,
 
   // Global page headers: https://go.nuxtjs.dev/config-head
   head: {
-    titleTemplate: '%s - test-blueprint',
-    title: 'test-blueprint',
+    titleTemplate: '%s - NuxT',
```
Comment (field006) [inline nuxt.config.js]: why change the title ?

File: nuxt.config.js [hunk 3]
```diff
@@ -41,14 +43,35 @@ export default {
...
   ],
 
-  // Axios module configuration: https://go.nuxtjs.dev/config-axios
-  axios: {
-    // Workaround to avoid enforcing hard-coded localhost:3000: https://github.com/nuxt-community/axios-module/issues/308
-    baseURL: '/',
+  auth: {
+    strategies: {
+      local: {
+        // token: {
```
Comment (field006) [inline nuxt.config.js]: remove the line of code is not used

File: pages/services/productServices.js
Comment (RyeConstantine) [reply pages/services/productServices.js:1]: not used, so i delete it

File: layouts/default1.vue
Comment (RyeConstantine) [reply layouts/default1.vue:1]: its a backup layout so anything goes wrong we can revert back.

File: layouts/noheader.vue (hunk 1 above)
Comment (RyeConstantine) [reply layouts/noheader.vue]: alright, right away

File: nuxt.config.js (hunk 3 above)
Comment (RyeConstantine) [reply nuxt.config.js]: allright, right away

File: store/index.js
Comment (RyeConstantine) [inline store/index.js:1]: i added this because the error asked to add index in store

File: layouts/default1.vue
Comment (field006) [inline layouts/default1.vue:1]: remove this file 
we can use revert to revert the code

File: nuxt.config.js (hunk 2 above)
Comment (RyeConstantine) [reply nuxt.config.js]: forgot to change it back, i alrady change it back

File: layouts/default1.vue
Comment (RyeConstantine) [reply layouts/default1.vue:1]: done

**Pull request merged**